hazelcast-client/src/main/java/com/hazelcast/client/spi/impl/ClientMembershipListener.java

```diff
@@ -68,7 +68,7 @@
     }
 
     @Override
-    public void handleMemberEventV10(Member member, int eventType) {
+    public void handleMemberEventV10(com.hazelcast.core.Member member, int eventType) {
```

Comment:
Do we need included package name?

Reply:
For now yes, client protocol depends on `com.hazelcast.core.Member` interface. After this PR gets merged, we should fix the client protocol and then remove `com.hazelcast.core.Member` from codebase completely.